hack/make/validate-lint

```diff
@@ -73,11 +73,14 @@ packages=(
 	pkg/reexec
 	pkg/signal
 	pkg/sockets
+	pkg/stdcopy
 	pkg/streamformatter
 	pkg/stringid
 	pkg/stringutils
-	pkg/sysinfo
 	pkg/symlink
+	pkg/sysinfo
+	pkg/system
+	pkg/systemd
```

Comment:
Could you remove this one ? ``pkg/systemd`` is not here anymore (and thus is not covered by the PR :sweat_smile:)